tests/unit/components/sl-translate-test.js

```diff
@@ -60,32 +60,6 @@ test( 'DOM and content of rendered translation', function( assert ) {
     );
 });
 
```

Comment:
Thought the "DOM and content of rendered translation" test was also going to be removed?